sys/include/net/ng_netconf.h

```diff
@@ -119,8 +119,12 @@ typedef enum {
      * @note not all transceivers may support this interrupt
      */
     NETCONF_OPT_TX_END_IRQ,
-    NETCONF_OPT_AUTOCCA,            /**< en/disable to check automatically
-                                         before sending the channel is clear. */
+    NETCONF_OPT_CSMA,               /**< en/disable to check automatically
```

Comment:
> What do you think of renaming NETCONF_OPT_AUTOCCA to NETCONF_OPT_CCA and introduce NETCONF_OPT_CSMA as a new option. The kw2x device e.g. does not implement the option NETCONF_OPT_CSMA. This would allow to en/disable SW-CSMA in the MAC-layer according to the devices' capabilities.

I like that. But how to ensure that these options don't interfere? I could imagine to implement AUTOCCA for the at86rf2xx by setting minBE = maxBE = 0 and CSMA_RETRIES = 0. But then you shouldn't mess with the CSMA setting.

When implementing SW-CSMA, one would try enabling CSMA first, and if that fails, try AUTOCCA I guess.

What is the bahaviour of kw2xrf? Is it trying resend the packet when channel is not clear? Anyway we need signaling if the packet was actually sent. I have an implementation for that, but didn't PR yet.